apple-pie/index.html

```diff
@@ -4,18 +4,49 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link href="styles/style.css" type="text/css" rel="stylesheet"/>
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+
+    <header>
+      <h1 class='title'>Apple Pie</h1>
+    </header>
+
+    <section class="firstText">
+      <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+        be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
+        favorites as well!</p>
+      <img class="preparation-img" src="images/recipe-info.png" alt="Recipe-info">
+      <hr>
+    </section>
+
+    <section>
+      <h2 class="subtitle">Ingredients</h2>
+        <ul class="list-toppings">
+          <li>1 recipe pastry for a 9 inch double crust pie.</li>
+          <li>1/2 cup unsalted butter.</li>
+          <li>3 tablespoons all-purpose flour.</li>
+          <li>1/4 cup water.</li>
+          <li>3 tablespoons all-purpose flour.</li>
+          <li>1/4 cup water.</li>
+          <li>1/2 cup white sugar.</li>
+          <li>1/2 cup packed brown sugar.</li>
+          <li>8 Granny Smith apples - peeled, cored and sliced</li>
+        </ul>
+        <img class="preparation-img" scr="https://raw.githubusercontent.com/NachoLG22/lab-css-recipes-clone/master/apple-pie/images/cooking-info.png" alt=" Cooking-info">
```

Comment:
```suggestion
        <img class="preparation-img" src="images/cooking-info.png" alt=" Cooking-info">
```
Aqui el problema es que has puesto la ruta y si te fijas has puesto scr y no src xD suele pasar mucho no worries!!!